google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpc.java

```diff
@@ -230,6 +232,19 @@ public String getNextPageToken() {
   OperationFuture<Backup, CreateBackupMetadata> createBackup(
       com.google.cloud.spanner.Backup backupInfo) throws SpannerException;
 
+  /**
+   * Creates a copy backup from the source backup specified.
+   *
+   * @param destinationBackup the backup to create. The instance, database, and expireTime fields of
+   *     the backup must be filled. It may also optionally have an encryption config set. If no
+   *     encryption config has been set, the new backup will use the same encryption config as the
+   *     source backup.
+   * @return the operation that monitors the backup creation.
+   */
+  OperationFuture<Backup, CopyBackupMetadata> copyBackUp(
```

Comment:
This would be a breaking change. Change to:

```
default OperationFuture<Backup, CopyBackupMetadata> copyBackup(BackupId sourceBackupId, com.google.cloud.spanner.Backup destinationBackup) throws SpannerException {
  throw new UnsupportedOperationException("Unimplemented");
}
```